### PR TITLE
update

### PR DESCRIPTION
**Pull request by ghost**
added symlink to .zsh-theme
update color codes

### CODE REVIEW

File: arc.zsh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Arc
 
 PROMPT_SYMBOL='▲'
-EXIT_VALUE_SYMBOL="%{$fg_bold[magenta]%}△%{$reset_color%}"
+EXIT_VALUE_SYMBOL="%F{magenta}%B△%b%f"
```
Comment (opodartho) [inline arc.zsh:4]: @eendroroy Why we should use F instead of $fg ?

**Comment by ghost** [comment]
@opodartho 
to use without 'autoload -U add-zsh-hook'. it depends on predefined color variables.
but with basic syntax you can use any color code

**Comment by opodartho** [comment]
Thank you @eendroroy

**Comment by ghost** [comment]
Welcome @opodartho

**Pull request merged**